features/step_definitions/networking.rb

```diff
@@ -975,3 +975,12 @@
   raise "Failed to kill the #{ovndb} database daemon" unless @result[:success]
 end
 
+Given /^OVN is functional on the cluster$/ do
+  ensure_admin_tagged
+  desired_ovnkube_node_replicas = daemon_set('ovnkube-node', project('openshift-ovn-kubernetes')).replica_counters(user: admin)[:desired]
+  available_ovnkube_node_replicas = daemon_set('ovnkube-node', project('openshift-ovn-kubernetes')).replica_counters(user: admin)[:available]
+  desired_ovnkube_master_replicas = daemon_set('ovnkube-master', project('openshift-ovn-kubernetes')).replica_counters(user: admin)[:desired]
+  available_ovnkube_master_replicas = daemon_set('ovnkube-master', project('openshift-ovn-kubernetes')).replica_counters(user: admin)[:available]
```

Comment:
does `daemon_set` cache results?  or can we use temp variables to avoid calling the function multiple times?

or maybe we can use `.values_at` on the hash.

```ruby
  ovnkube_node_ds = daemon_set('ovnkube-node', project('openshift-ovn-kubernetes')).replica_counters(user: admin)
  desired_ovnkube_node_replicas, available_ovnkube_node_replicas = ovnkube_node_ds.values_at(:desired, :available)

```

Reply:
hmm, i have to check on that Ross. Thanks

Reply:
Shortened it via temp variables and values_at array makes sense, thanks